doc/lspconfig.txt

```diff
@@ -129,50 +87,6 @@ All `config` elements described in `:help vim.lsp.start_client()` can
 additionally be overridden via the `setup {}` call. The most commonly
 passed overrides to `setup {}` are:
 
-- {capabilities} `table <string, string|table|bool|function>`
-
-  a table which represents the neovim client capabilities. Useful for
-  broadcasting to the server additional functionality (snippets, off-protocol
-  features) provided by plugins.
-
-- {cmd} `list[string]`
-
-  a list where each entry corresponds to the blankspace delimited part of
-  the command that launches the server. The first entry is the binary used
-  to run the language server. Additional entries are passed as arguments.
-
-  The equivalent `cmd` for: 
->
-  foo --bar baz
-<
-  is:
->
-  {'foo', '--bar', 'baz'}
-<
-- {handlers} `list[functions]`
-
-  a list of handlers which override the function used to process a response
-  from a given language server. Applied only to the server referenced by
-  setup. See |lsp-handler|.
-
-- {init_options} `table <string, string|table|bool>`
-
-  a table passed during the initialization notification after launching
-  a language server. Equivalent to the `initializationOptions` field found
-  in `InitializeParams` in the LSP specification.
-
-  See upstream server documentation for available initialization
-  options.
-
-- {on_attach} `function(client, bufnr)`
-
-  Callback invoked by Nvim's built-in client when attaching a buffer to a
-  language server. Often used to set Nvim (buffer or global) options or to
-  override the Nvim client properties (`server_capabilities`) after a
-  language server attaches.
-
-  Prefer using an |LspAttach| autocommand handler instead.
-
 - {settings} `table <string, string|table|bool>`
```

Comment:
except this one looks rather useful. probably should move this into `:help lsp` ...